frontends/election-manager/src/components/smartcard_modal/status_message.tsx

```diff
@@ -126,28 +126,25 @@ export function InProgressStatusMessage({
   actionStatus,
 }: InProgressStatusMessageProps): JSX.Element | null {
   const { action } = actionStatus;
-  const actionRoleReadableString = userRoleToReadableString(actionStatus.role);
 
   let text: string;
   switch (action) {
     case 'Program': {
-      text = `Creating ${actionRoleReadableString} Card`;
+      text = 'Programming card';
       break;
     }
     case 'PinReset': {
-      text = `Resetting ${actionRoleReadableString} Card PIN`;
+      text = 'Resetting card PIN';
       break;
     }
     case 'Unprogram': {
-      text = `Unprogramming ${actionRoleReadableString} Card`;
+      text = 'Unprogramming card';
```

Comment:
We previously capitalized "Card" here because every other word in the line was capitalized, e.g. "Unprogramming Election Manager Card", and it looked a bit funny to have a single uncapitalized word at the end of the string (https://github.com/votingworks/vxsuite/pull/2267#discussion_r937251167). For consistency, I ended up capitalizing "Card" following capitalized user roles throughout this modal.

Now that we don't have a user role name here, lower case works fine. And I've reverted to using lower case "card" even after capitalized user role names elsewhere since that's what we do in _most_ places across our UIs.